<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-BJLM547G" failures="2" tests="2" name="Test(failed)" time="0.046" errors="0" timestamp="2023-12-14T22:28:07 IST">
  <testcase classname="RetryLogicDemo.TestCasesExample" name="TestCase01" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RetryLogicDemo.TestCasesExample.TestCase01(TestCasesExample.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TestCase01 -->
  <testcase classname="RetryLogicDemo.TestCasesExample" name="TestCase02" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RetryLogicDemo.TestCasesExample.TestCase02(TestCasesExample.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TestCase02 -->
  <testcase classname="RetryLogicDemo.TestCasesExample" name="TestCase03" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase03 -->
</testsuite> <!-- Test(failed) -->
